assets/templates/header.html

```diff
@@ -46,13 +46,13 @@
     <li class="nav-item"><a href="forgot-password.html" class="nav-link">Forgot</a></li>
   </ul><!-- nav -->
 </section><!-- section-module -->
-    <header class="header-main">
+    <header class="header-main header-is-active">
       <div class="header-wrapper">
         <div class="logo"><span class="ms-icon icon-doppler-logo"></span></div>
         <nav class="nav-left-main">
           <div class="menu-main--container">
             <ul class="menu-main">
-              <li class="submenu-item"><a href="#">Campañas</a>
+              <li class="submenu-item"><a href="#" class="active">Campañas</a>
                 <ul class="sub-menu sub-campaings">
                   <li><a href="#">Borradores</a></li>
```

Comment:
the `active` class is missing

Reply:
Estaría bien, para tener un ejemplo más realista.

Reply:
Estoy aplicando este cambio